app/templates/public.hbs

```diff
@@ -12,6 +12,9 @@
           <div class="one wide column"></div>
           <div class="fifteen wide column">
             <h4 class="event time">{{header-date model.startsAt model.timezone}}</h4>
+            {{#if displayEndDate}}
+              <h5 class="event time ends">{{t 'To'}} {{header-date model.endsAt model.timezone}}</h5>
```

Comment:
@kushthedude why aren't you using same class and font size as that of eventStartDate? Any specific reason to male the eventEndDate smaller?

Reply:
@shreyanshdwivedi mainly for two reasons
* Fir devices with small screen , same size font is looking ugly and causing an underflow.
* Since in the ‘end date’ there is an extra ‘to’ and if i make it of same size as of start date , second row will have more width than first row which dont looks good